src/test/java/teammates/test/cases/datatransfer/FeedbackSessionAttributesTest.java

```diff
@@ -13,61 +13,21 @@
 import teammates.common.util.TimeHelper;
 import teammates.test.cases.BaseTestCase;
 
-import com.google.appengine.api.datastore.Text;
-
 public class FeedbackSessionAttributesTest extends BaseTestCase {
-    private static String feedbackSessionName;
-    private static String courseId;
-    private static String creatorId;
-    private static Text instructions;
-    private static Date createdTime;
-    private static Date startTime;
-    private static Date endTime;
-    private static Date sessionVisibleFromTime;
-    private static Date resultsVisibleFromTime;
-    private static double timeZone;
-    private static int gracePeriod;
-    private static FeedbackSessionType feedbackSessionType;
-    private static boolean sentOpenEmail;
-    private static boolean sentClosingEmail;
-    private static boolean sentClosedEmail;
-    private static boolean sentPublishedEmail;
-    private static boolean isOpeningEmailEnabled;
-    private static boolean isClosingEmailEnabled;
-    private static boolean isPublishedEmailEnabled;
+    private Date startTime;
+    private Date endTime;
     
-    private static FeedbackSessionAttributes fsa;
+    private FeedbackSessionAttributes fsa;
     
     @BeforeClass
     public void classSetup() {
-        feedbackSessionName = null;
-        courseId = null;
-        creatorId = null;
-        instructions = null;
-        createdTime = null;
         startTime = TimeHelper.combineDateTime("09/05/2016", "1000");
         endTime = TimeHelper.combineDateTime("09/05/2017", "1000");
-        sessionVisibleFromTime = null;
-        resultsVisibleFromTime = null;
-        timeZone = 8;
-        gracePeriod = 15;
-        feedbackSessionType = FeedbackSessionType.STANDARD;
-        sentOpenEmail = false;
-        sentClosingEmail = false;
-        sentClosedEmail = false;
-        sentPublishedEmail = false;
-        isOpeningEmailEnabled = false;
-        isClosingEmailEnabled = false;
-        isPublishedEmailEnabled = false;
         
-        fsa = new FeedbackSessionAttributes(feedbackSessionName,
-                courseId, creatorId, instructions,
-                createdTime, startTime, endTime,
-                sessionVisibleFromTime, resultsVisibleFromTime,
-                timeZone, gracePeriod, feedbackSessionType,
-                sentOpenEmail, sentClosingEmail, sentClosedEmail, sentPublishedEmail,
-                isOpeningEmailEnabled, isClosingEmailEnabled, isPublishedEmailEnabled
-                );
+        fsa = new FeedbackSessionAttributes(null,
+                null, null, null, null, startTime, endTime,
+                null, null, 8, 15, FeedbackSessionType.STANDARD,
+                false, false, false, false, false, false, false);
```

Comment:
Yes, but that doesn't help much in a case like this when there are many parameters. Intellij has taken it to a whole new level:
![image](https://cloud.githubusercontent.com/assets/1673303/23285822/6d13e304-fa6e-11e6-913e-7af3f2e39665.png)